sync/awsdatabricks.py

```diff
@@ -808,7 +812,7 @@ def _get_eventlog(cluster_description: dict, run_end_time_millis: int) -> Respon
         )
 
 
-def _get_all_cluster_events(cluster_id: str):
+def get_all_cluster_events(cluster_id: str):
```

Comment:
Made this public so I could save the full cluster events.

@gseyffert-sync I'm curious if it was a deliberate decision to not use this method in the `get_cluster_report` method. It seems the current state of that method will not return the full cluster if it is over a certain number of events.

Reply:
ah, this was actually just a miss - the intent is definitely to use this method (originally I had this logic in the `awsdatabricks` client method, but didn't update that call site when I added this function).

Reply:
If we just fix `get_cluster_report`, does this function still need to be public?

Reply:
Nope, I'll make that change and make this method private again.